(venv) PS C:\Users\pcdan\Licita_Facil\LicitAI\backend> uvicorn main:app --reload
INFO:     Will watch for changes in these directories: ['C:\\Users\\pcdan\\Licita_Facil\\LicitAI\\backend']
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [15804] using WatchFiles
Process SpawnProcess-1:
Traceback (most recent call last):
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\asyncio\base_events.py", line 687, in run_until_complete       
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string   
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pcdan\AppData\Local\Programs\Python\Python312\Lib\importlib\__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\main.py", line 7, in <module>
    from src import crud, models, schemas, analysis_service
  File "C:\Users\pcdan\Licita_Facil\LicitAI\backend\src\analysis_service.py", line 50
    finally:
    ^^^^^^^
SyntaxError: invalid syntax
